Now with 200% more brrt

Sigil9963 opened the pull request:
Updated Vulcan sound effects
Adjusted animation timings on secondary weapons.

Commit:
Now with 200% more brrt

Updated Vulcan sound effects
Adjusted animation timings on secondary weapons.

Sigil9963 commented:
Also disabled stagger/knockback on explosive weapons

Commit:
Replaced Slug Rifle sounds with later build versions.

TimeMaster18 commented:
I think it would be better to move this one on a different file. 
If the original file it gets too big that is hard to use / read, it might be worth start splitting it by any trait or feature you see fit.

Commit:
Minor hex corrections. Added icon for Radar item.
Split hex edits into core gameplay and audio files. Audio can be an optional patch for those who want the updated sounds.

Commit:
Correct turret mode speed hex address.

Commit:
Changed Night Pools to Red Camo. Changed Bubblegum Prime to Purple Camo. Fixed Blockade spawn rotation.
